src/FunctionApps/python/IntakePipeline/__init__.py

```diff
@@ -5,42 +5,93 @@
 from IntakePipeline.transform import transform_bundle
 from IntakePipeline.linkage import add_patient_identifier
 from IntakePipeline.fhir import (
-    read_fhir_bundles,
     upload_bundle_to_fhir_server,
     store_bundle,
+    store_message,
     get_fhirserver_cred_manager,
 )
+from IntakePipeline.conversion import (
+    convert_batch_messages_to_list,
+    convert_message_to_fhir
+)
 
 from config import get_required_config
 
 from phdi_transforms.geo import get_smartystreets_client
 
 
-def run_pipeline():
+def run_pipeline(blob: func.InputStream, filetype: str):
     salt = get_required_config("HASH_SALT")
     geocoder = get_smartystreets_client(
         get_required_config("SMARTYSTREETS_AUTH_ID"),
         get_required_config("SMARTYSTREETS_AUTH_TOKEN"),
     )
+    fhir_url = get_required_config("FHIR_URL")
     fhirserver_cred_manager = get_fhirserver_cred_manager(
-        get_required_config("FHIR_URL")
+        fhir_url
     )
 
     container_url = get_required_config("INTAKE_CONTAINER_URL")
-    container_prefix = get_required_config("INTAKE_CONTAINER_PREFIX")
-    output_path = get_required_config("OUTPUT_CONTAINER_PATH")
+    valid_output_path = get_required_config("VALID_OUTPUT_CONTAINER_PATH")
+    invalid_output_path = get_required_config("INVALID_OUTPUT_CONTAINER_PATH")
+
+    if filetype == "elr":
+        message_type = "oru_r01"
+        message_format = "hl7v2"
+    elif filetype == "vxu":
+        message_type = "vxu_v04"
+        message_format = "hl7v2"
+    elif filetype == "eICR":
+        message_type = "ccda"
+        message_format = "ccda"
+    else:
+        raise Exception(f"Found an unidentified message_format: {filetype}")
 
-    for bundle in read_fhir_bundles(container_url, container_prefix):
-        transform_bundle(geocoder, bundle)
-        add_patient_identifier(salt, bundle)
-        store_bundle(container_url, output_path, bundle)
-        upload_bundle_to_fhir_server(fhirserver_cred_manager, bundle)
+    # The HL7 messages that VA sent have weird unicode characters at the beginning
+    # and end of the files (\\u000b & \\u001c, respectively) that can't be decoded.
+    # The quick fix is to ignore the errors.
+    # TODO: Properly handle the erroneous unicode characters.
+    messages = convert_batch_messages_to_list(
+        blob
+        .read()
+        .decode("utf-8", errors="ignore")
+    )
+    for i, message in enumerate(messages):
+        token = fhirserver_cred_manager.get_access_token()
+        response = convert_message_to_fhir(
+            message=message,
+            message_format=message_format,
+            message_type=message_type,
+            access_token=token.token,
+            fhir_url=fhir_url
+        )
+        if response.status_code == 200:
+            bundle = response.json()
+            transform_bundle(geocoder, bundle)
+            add_patient_identifier(salt, bundle)
+            store_bundle(container_url, valid_output_path, bundle)
```

Comment:
This method requires the bundle type to be supplied (elr, ecr, or vxu).  These values come from constants defined in the GenerateCSV Azure function and differ (but have a 1-1 relationship with) the filetype paramater.  These should probably be standardized across the solution.  In the short term, I think we'll need to pass either one of those values to this function - probably derived from the filetype parameter.  

Code that uses type:
https://github.com/CDCgov/prime-public-health-data-infrastructure/blob/6152260fe4e6f033d0e500fdcb4e587d478aa4e6/src/FunctionApps/python/IntakePipeline/__init__.py#L33